src/addAllAssetsToCompilation.js

```diff
@@ -31,7 +31,16 @@ function resolveOutput(compilation, addedFilename, outputPath) {
 async function addFileToAssets(
   compilation,
   htmlPluginData,
-  { filepath, typeOfAsset = 'js', includeSourcemap = true, hash = false, publicPath, outputPath, files = [] }
+  {
+    filepath,
+    typeOfAsset = 'js',
+    includeSourcemap = true,
+    includeGzipped = true,
```

Comment:
Nah, it errors: https://github.com/jantimon/html-webpack-plugin/blob/master/index.js#L307-L313

We could do something like `addRelated: true` as an alternative to the array I proposed in the issue, which would use a glob do do ```const extraFiles =`${filename}.*`;``` and see what if can find. WDYT?